<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsTransformSettingsDialog</class>
 <widget class="QDialog" name="QgsTransformSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>356</width>
    <height>381</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transformation settings</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Transformation type:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cmbTransformType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="textLabel1">
       <property name="text">
        <string>Resampling method:</string>
       </property>
       <property name="buddy">
        <cstring>cmbResampling</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="cmbResampling">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <item>
        <property name="text">
         <string>Nearest neighbour</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Linear</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cubic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Cubic Spline</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lanczos</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="mCompressionLabel">
       <property name="text">
        <string>Compression:</string>
       </property>
       <property name="buddy">
        <cstring>cmbCompressionComboBox</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="cmbCompressionComboBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>NONE</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>LZW</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>PACKBITS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>DEFLATE</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Output raster:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLineEdit" name="leOutputRaster"/>
       </item>
       <item>
        <widget class="QToolButton" name="tbnOutputRaster">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Target SRS:</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLineEdit" name="leTargetSRS"/>
       </item>
       <item>
        <widget class="QToolButton" name="tbnTargetSRS">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Generate pdf report:</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLineEdit" name="mReportFileLineEdit"/>
       </item>
       <item>
        <widget class="QToolButton" name="tbnReportFile">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="7" column="0" colspan="2">
      <widget class="QCheckBox" name="cbxUserResolution">
       <property name="text">
        <string>Set Target Resolution</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Horizontal</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QgsValidatedDoubleSpinBox" name="dsbHorizRes">
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>0.000000000000000</double>
       </property>
       <property name="maximum">
        <double>999999.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Vertical</string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QgsValidatedDoubleSpinBox" name="dsbVerticalRes">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="decimals">
        <number>5</number>
       </property>
       <property name="minimum">
        <double>-999999.000000000000000</double>
       </property>
       <property name="maximum">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QCheckBox" name="mWorldFileCheckBox">
       <property name="text">
        <string>Create world file</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="cbxZeroAsTrans">
     <property name="text">
      <string>Use 0 for transparency when needed</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QCheckBox" name="cbxLoadInQgisWhenDone">
     <property name="text">
      <string>Load in QGIS when done</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsValidatedDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsvalidateddoublespinbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbTransformType</tabstop>
  <tabstop>cmbResampling</tabstop>
  <tabstop>cmbCompressionComboBox</tabstop>
  <tabstop>leOutputRaster</tabstop>
  <tabstop>tbnOutputRaster</tabstop>
  <tabstop>leTargetSRS</tabstop>
  <tabstop>tbnTargetSRS</tabstop>
  <tabstop>cbxUserResolution</tabstop>
  <tabstop>dsbHorizRes</tabstop>
  <tabstop>dsbVerticalRes</tabstop>
  <tabstop>cbxZeroAsTrans</tabstop>
  <tabstop>cbxLoadInQgisWhenDone</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsTransformSettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsTransformSettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
